R/descr.r

```diff
@@ -1,4 +1,6 @@
 descr <- function(variable,na=NA, p.weights = NULL, na.rm = FALSE, verbose=TRUE) {

```

Comment:
what kind of input is needed here?
variable: I can check for data.frame, but then e.g. numeric vectors are not accepted; and the other way around.
na: should I check for something or just leave it be?

Reply:
I'm sorry for the possible confusion. `variable` may be either a (numeric) variable or a `data.frame` with several variables. If `variable` is a `data.frame`, the function gives descriptives for all numerical variables, omitting non-numeric ones. 
I think the suggested assertions work well. No other ones are necessary.